Library/Homebrew/dev-cmd/extract.rb

```diff
@@ -83,6 +83,7 @@ module Homebrew
   sig { returns(CLI::Parser) }
   def extract_args
     Homebrew::CLI::Parser.new do
+      usage_banner "`extract` [<options>] <formula> <destination_tap>"
```

Comment:
```suggestion
      usage_banner "`extract` [<--version>`=`] [<--force>] <formula> <destination_tap>"
```

For consistency with other commands. If this is too long to fit on one line in `brew help extract`, though, the `[<options>]` line is fine.

Reply:
If I make the last arg `tap` then it's fine.

```
➜ brew help extract
Usage: brew extract [--version=] [--force] formula tap
```

Reply:
No strong feelings I guess. I'm happy to leave it as-is. If it were me, I'd probably just go with `tap`. I think if the user is confused they should be reading the description (they shouldn't be running `brew extract` if they don't know what it does) which makes it clear.